omod/src/main/web/dashboardwidgets/appointments/appointments.html

```diff
@@ -0,0 +1,8 @@
+<ul>
+    <li ng-if="$ctrl.appointments.length != 0" ng-repeat="appointment in $ctrl.appointments track by $index">
+      <div >{{appointment.date| date:'mediumDate'}}   {{appointment.startTime | date:'h:mm a'}} - {{appointment.endTime | date:'h:mm a'}} {{appointment.ServiceType  | translate }}</div>
```

Comment:
For consistency can we please change the format slightly.  
`      <div >{{appointment.date| date:'dd.MMM.yyyy'}},  {{appointment.startTime | date:'h:mm a'}} - {{appointment.endTime | date:'h:mm a'}}, {{appointment.ServiceType  | translate }}</div>
`

Reply:
Alright

Reply:
Thanks

Reply:
done